src/main/translations/fr.translation-meta.xml

```diff
@@ -74,4 +74,8 @@
     <label>Abandonner la demande</label>
     <name>cookbook_AbandonRequest</name>
   </customLabels>
+  <customLabels>
+    <label>créez la demande</label>
```

Comment:
Shouldn't it be `Soumettre la demande` instead?

Reply:
In any case it should be `créer la demande` ;)

Reply:
@acarpentier01  Please help us choose the right translation for the label of this "Create case" button.

Reply:
@mmitiche How about: "Créez une demande"
Also, instead of "Abandonner la demande"-->"Abandonnez votre demande"